app/views/devise/sessions/new.html.erb

```diff
@@ -1,63 +1,53 @@
 <div class="page-header">
-  <h1>Login into Mezuro</h1>
+  <h1><%= t('.sign_in', :default => "Sign in") %></h1>
 </div>
 
-<%= form_for(resource, :as => resource_name, :url => session_path(resource_name),:html => { :class => "form-inline" }) do |f| %>
+<%= form_for(resource, :as => resource_name, :url => session_path(resource_name), :html => { :class => "form-inline" }) do |f| %>
   <div class="row margin-left-none">
     <div class="form-table col-md-9">
       <div class="form-row">
         <div class="field-container">
-          <%= f.label :email, class: "control-label " %>
-          <%= f.email_field :email, autofocus: true, tabindex: 1, class: 'text-field form-control' %>
+          <%= f.label :email, class: 'control-label' %><br />
+          <%= f.email_field :email, :autofocus => true, class: 'text-field form-control' %>
```

Comment:
Is necessary if the user wants to navigate using tabs after the page was loaded.